### ISSUE TITLE
Suggestion: more nuance in the fundraising rule

### ISSUE THREAD

**Issue by bwyatt** [hachyderm/community#401]
### What is your suggestion?

_I apologize if this is duplicative of other work/discussions, but I haven't found anything in the repo._

I'd like to see more nuance applied to the fundraising rule that was established under #37. 

I just migrated to Hachyderm from another instance (tech.lgbt), and one of my mutuals from there expressed some concern about Hachyderm's approach to fundraising, which I hadn't seen before moving my account. (See [this post](https://tech.lgbt/@cariad/110259155856772374).)

I can understand the restriction against accounts solely dedicated to fundraising, but I would like to see some room allowed for folks to raise funds for personal needs, perhaps in narrowly-defined scenarios. 

We unfortunately live in a world where fundraising may be required for individuals to access healthcare or to ensure their personal safety, and outright banning of posts seeking this type of help feels like asking members not to bring their whole selves to Hachyderm.

Perhaps we should allow fundraising _posts_ with certain caveats:
- Only on certain account types
- Only for specific purposes (e.g., funds to secure healthcare or personal safety)
- A limit on frequency (maybe a # of posts per day, # of posts ever, or refer back to spam guidelines)
- A limit on account age (e.g., must be here 90 days & posting actively on other topics)

**Comment by quintessence** [comment]
Thank you for opening this issue! This is something we're discussing this week and would love to see / hear some thoughts about it. So some of the nuance:

* Rules we modify / introduce / etc. need to be easy for the community to understand and do
  * As an extreme example, it's like installing new software with a lengthy TOS that users _generally_ don't read and just click Accept. This is because the process itself introduces a lot of friction in an attempt to be clear.

We structured the bot rules as similar. We tried to make the bot account type something that passes the "glance test". As in, someone can tell "at a glance" if the bot they want to write would fall in the rules or not.

The discussions we're having around the clarifications we'll publish for the anti-spam policy are going to need to be similar. We want Hachydermians to be able to tell, at a glance, if they can or can't do something. 

Some additional nuance:

* Rule creation for Hachyderm is impact-focused.

The reason the bot rule has the posting limit is to prevent the impact of having bots that (intentionally or accidentally) post often enough that the Local Timeline is no longer useful. Similarly, all bots need to be summoned to prevent the impact of bots that engage with users without their consent (i.e. the summon).

Pivoting all of that to the issue at hand: a common source of spam are GoFundMe (and similar) links generated by profiles that mimic genuine human distress (whether they're automated or not). In terms of impact, what we're working on this week and the impacts that we're using as constraints:

* **Hoaxes negatively impact people's ability to request mutual aid.**<br />
This aspect of our anti-spam measures is to prevent the impact of a timeline flooded with disingenuous, bot or bot-like, profiles and bids for funding. Another impact of an overrun timeline is that it would _also_ prevent genuine requests for aid, as it would pollute potential giver's ability to trust that the account posting for aid is a person and not a hoax.
* **We do not want to trigger trauma by asking people to justify their circumstances.**<br />
We would not want to create the impact of "why" questions. To put it another way: we don't want to say that some are more deserving of aid than others because "these people need groceries", "these other people need rent", "these other people are about to foreclose. One person's inconvenience is another person's emergency, for a full set of systemic reasons.
* **We do not want to create a situation where moderation issues go unresolved for long periods of time.**<br />
Essentially: we don't want a situation where someone is reported for posting for mutual aid "too early". Crises happen as they appear.

I suppose a more succinct way to say all of that is: we do not, and will not, create a situation where the rules have the impact of making a Neighborhood Watch scenario where people are too afraid to ask for help because people are looking at them, their accounts, their posts and seeing if they're In Alignment and then reporting them for "putting a toe out of line". The crisis itself is emergency enough.

The impact we do want to create:

* **Prevent hoaxes from appearing as spam, mitigate the impact to requests for mutual aid.**<br />
Basically: we still see this as a sort of spam at times and will need to handle, but we want to make sure that people in a crisis still feel comfortable and confident to 1 ) ask for help or 2 ) notify us if there was an error.
* **Create a situation where Hachydermians respond with love and care.**<br />
Basically: we do not want to create rules for Hachydermians to enforce / report to us for users falling out of line with in this situation. We want to create rules that encourage people to respond with nurturance and care.

I would personally welcome any and all community discussion here while we, the moderation team, deliberate over this this week.

**Comment by krisnova** [comment]
Hey everyone -- I'm very busy these days and haven't had as much as I would have liked to lead us through these types of situations. 

Let me start off by saying unequivocally that Hachyderm supports all calls for mutual aid. Period. 

This was never in question in my mind, nor do we have any intention of turning people away who need mutual aid.

I wish I could share more of my life here, however here is a small anecdote that hopefully captures where I am with this. Personally I spent roughly 8 years of my life unhoused and dependent on mutual aid for survival. The topic of passing wealth and fortune off to others is very dear to my heart. I have lost another trans woman who I deeply loved to suicide. In her specific scenario she would likely be here if she had more support during her time of need.

We will ABSOLUTELY update the rules and our wording, and will make it clear that we support any marginalized person in need. 

I think we can find other ways of dealing with spam. I believe we have some ideas in a potential future Nivenly project to contribute some features to open source to help surface bot accounts and so forth. The point I'm trying to make is that our quest to reduce spam should never come at the expense of silencing someone in need. We can (and will) do better. 

Thanks for surfacing this concern! Hachyderm and Nivenly are intended to be democratic organizations and this is a great example of using the process to instill change for the better! We are working as fast as we can to create better tools for faster feedback loops from our users and members. 

Hope this helps. We can close this issue as soon as the docs are updated to reflect this important change. 

Thanks again ✌🏻

**Comment by ThisIsMissEm** [comment]
Would it be worth specifying that solicitation of mutual aid should just be posts & not direct engagement attempts? (E.g., sliding into someone's mentions to solicit mutual aid is rather presumptive/rude & places individuals in awkward position, but posting publicly on a hashtag is okay, as with boosting posts seeking aid.

The only reason I mention this is because directly targeting specific people for mutual aid is both spammy and awkward (of course you can reply with a link if someone asks for it, but y'know)

**Comment by bwyatt** [comment]
Thanks all. I realize now looking back at my post that I wrote it too hastily and definitely argued for a more restrictive rule than I intended. I appreciate you all embracing the spirit of the request and working to make sure mutual aid posts are allowed and respected on Hachyderm. You're building a great community here, and I’m happy to take part in it. ❤️

**Comment by quintessence** [comment]
Just to follow up quickly: you did perfect. The post is the perfect request and the fact that the discussion is lengthy isn't a reflection of anything you did incorrectly. This is the right discussion to have, and you started it by making a clear request with the sources of your concern and the changes you'd like to see. None of the subsequent discussion about what nuance to apply is in any reflection a way that you didn't do enough or do it correctly.

For the discussion itself: I'm mostly staying out of it for a few days to allow for people to share their thoughts, opinions, and ideas :) I just wanted to respond to the initial request with some of the constraints we use when making and changing rules to help with the sharing of those ideas :)

**Comment by timdmackey** [comment]
I think the rule on fundraising urgently needs to be rewritten or completely expunged. As it stands it's incredibly restrictive, and I have a hard time believing that it was intended to be as restrictive as it is. I’m hoping it’s just a mistake!

This post ended up being longer than I had planned, but I think this is a really important topic. I hope you’ll have the patience to read this! Thank you all above for your thoughtful input. Hachyderm should be a welcome home for all, and I believe mutual aid is an essential part of that. 

The rule about fundraising occurs under the **"Disallowed Specialized Account Categories"** heading, but it includes verbiage which makes it apply to all posts by any account:
> The following are disallowed as general content on the server [...] Fundraising - No accounts can fundraise on Hachyderm.

If you take this rule as written, the recent complaints against hachyderm are completely valid. As most people would read it, mutual aid is a type of fundraising. That being so, this rule completely bans **any** requests for mutual aid, of any kind, in any circumstance. Surely this was not the intention? It’s not at all in line with the values of hachyderm as I’ve understood them.

Which leaves the question—why is this line even there at all? What type of “fundraising” is this meant to address? What type of fundraising is enough of a problem that it even needs to be addressed?

I’m my personal opinion, the rules around personal fundraising should be as permissive as humanly possible. As such, although I know they are well meaning, I’d disagree with  most of these proposals:

> Only for specific purposes (e.g., funds to secure healthcare or personal safety)

Who are we to say what constitutes a legitimate reason to be in need of money and to request help? If we limit fundraising to only specific purposes, it’s easy to imagine circumstances coming up that the rule author never thought of.

If certain categories of “fundraising” need to be excluded, then that’s another question of what they are and how necessary it really is to explicitly ban them.

> A limit on frequency (maybe a # of posts per day, # of posts ever, or refer back to spam guidelines)

I’m hesitant to put any hard limit on permitted frequency. “Be respectful” and “don’t be spammy” are enough in my mind. The generic spam rules would cover that I'd think. 

I’m absolutely against a limit on number of posts ever. Many people find themselves stuck in situations where they are in need for a very long time, and they should be offered the same grace as others who only need help during a short period.

> A limit on account age (e.g., must be here 90 days & posting actively on other topics)

I think a limit on account age would do more harm than good. I'm guessing the intention is to ensure that they're a real person, but there are other ways to do that? It's completely legitimate for a new user here to be in need the moment they join. After all, we're still welcoming migrants from the fiasco that is Twitter.

For the "posting actively on other topics” part, see below. 

> Only on certain account types
This I'd agree with. In my opinion, the only restriction on people fundraising (assuming we’re talking about mutual aid and not VCs) should be that the account is a “user/general” account. I.e., the account represents a person, it’s not a bot, and the person interacts with other people. I don’t think there should be or needs to be an explicit rule on what kind of other posts the person is making. As long as they’re acting like a regular human being, that should be enough. If the account seems scammy and someone reports it, then the mods can communicate with the account and use their judgement about whether the account is a person acting on good faith.

**Comment by bwyatt** [comment]
Hi @timdmackey! I agree with basically everything you said about my suggestions. As I said in another reply, I wrote the initial post hastily and advocated for some pretty unreasonable restrictions. 

There's an active pull request with an actual proposed rule change. It hasn't been linked here yet for some reason, so I'll do that now: #404 

I feel a bit out of my depth with this discussion since I've not personally been in a situation where I've needed to rely on mutual aid, and I'd rather leave the rule making to folks with more insight than I. Hopefully adding this link will help more folks find and participate in the discussion. 

https://github.com/hachyderm/community/pull/404

**Comment by quintessence** [comment]
I just realized this is still open, even though the updated language for the Monetary Policy has been live for a month now. Apologies! Closing :)
